python_modules/dagster/dagster_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -748,7 +748,9 @@ def test_event_log_step_stats(
         assert len(c_stats.attempts_list) == 1
 
     def test_secondary_index(self, storage: EventLogStorage):
-        if not isinstance(storage, SqlEventLogStorage):
+        if not isinstance(storage, SqlEventLogStorage) or isinstance(
+            storage, InMemoryEventLogStorage
+        ):
```

Comment:
with the other changes, do you need to skip these tests with in memory? if so why?

Reply:
ah yes I think this predated the has_secondary_index override, one moment

Reply:
ok I think it's the

assert not storage.has_secondary_index("_A")

test cases that would break - not clear to me that it's useful to change InMemoryEventLogStorage to diligently check which secondary indices it has or hasn't applied in mock format (when it just applies all of them now)